Assets/NuGet/Editor/NugetConfigFile.cs

```diff
@@ -76,6 +77,14 @@ public void Save(string filepath)
                     addElement.Add(new XAttribute("value", "true"));
                     disabledPackageSources.Add(addElement);
                 }
+
+                if (source.HasPassword)
+                {
+                    addElement = new XElement("add");
```

Comment:
Josh noticed that this config file is a standard. And this code here isn't the right format. This should be saved in packageSourceCredentials instead. I'll make that change.

Reply:
Fixed.